aspnet/migration/rc1-to-rc2.rst

```diff
@@ -17,7 +17,7 @@ For information regarding a complete list of breaking changes in RC2 for ASP.NET
 Creating your web application host
 ----------------------------------
 
-Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main`` that sets up a web host, then tells it to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
+Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main()`` that sets up a web host, then tells it to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
```

Comment:
I read the sentence with **you** being the subject. As in you must define ... then you tell it to start listening. It's awkward phrasing and splitting it into two sentence is much better.